libcaf_core/caf/detail/format.cpp

```diff
@@ -309,7 +309,7 @@ void copy_verbatim(ParserState& ps, copy_state& cs);
 template <class ParserState>
 void copy_formatted(ParserState& ps, copy_state& cs) {
   cs.reset();
-  auto guard = make_scope_guard([&] {
+  auto guard = make_scope_guard([&]() noexcept {
     if (ps.code <= pec::trailing_character)
       cs.fn = copy_verbatim<string_parser_state>;
```

Comment:
Looks like I didn't spot this last time. Please move below `fin()` to be consistent with the other recent changes and drop the include for `scope_guard`.